pkg/runtime/security/token.go

```diff
@@ -0,0 +1,15 @@
+package security
+
+import (
+	"os"
+)
+
+/* #nosec */
+const (
+	APITokenEnvVar = "DAPR_API_TOKEN"
```

Comment:
please add comments on the exposed constants for readability.

Reply:
done